File: patterns/c-patterns/pattern14.c
// Pattern 14: ...

// 55555
// 4444*
// 333**
// 22***
// 1****

#include <stdio.h>

int main()
{
    int i, j;
    int n = 5;
    for (i = n; i >= 1; i--)
    {
        for (j = 1; j <= n; j++)
        {
            if (j > i)
                printf("*");
            else
                printf("%d", i);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern14.js
// Pattern 14: ...

// 55555
// 4444*
// 333**
// 22***
// 1****

const n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';
    
    for (let j = 1; j <= n; j++) {
        if (j > i) {
            row += '*';
        } else {
            row += j;
        }
    }
    
    console.log(row);
}
